src/common/config/config.testnet.json

```diff
@@ -1,17 +1,13 @@
 {
-    "theme": {
-        "primaryColor": "#6200ee",
-        "secondaryColor": "#03dac4",
-        "tertiaryColor": "white",
-        "linkColor": "#0057B2"
-    },
-    "appLogoUrl": "/tonomy-logo48.png",
+    "themeFile": "theme.pangea.css",
+    "appLogoUrl": "/favicon.png",
     "appName": "Pangea testnet",
```

Comment:
appName: Pangea Testnet

or better would be "Pangea Passport - Testnet". But if you need to change all of the infrastructure again For this then don't worry